src/components/shared/ManagedTree.js

```diff
@@ -44,10 +44,6 @@ class ManagedTree extends Component {
       expanded: new Set(),
       focusedItem: null
     };
-
-    const self: any = this;
```

Comment:
The line defining the `self` constant I think is there to prevent Flow bugs? I know it's a pattern we use often when binding our methods.